test/test_insert_manager.rb

```diff
@@ -28,6 +28,25 @@ module Arel
         }
       end
 
+      it 'inserts multiple values' do
+        table = Table.new(:users)
+        manager = Arel::InsertManager.new
+        manager.into table
+
+        manager.columns << table[:id]
+        manager.columns << table[:name]
+
+        manager.values = manager.create_tuple([
+          %w{ 1 david },
+          %w{ 2 kirs },
+          ["3", Arel.sql('DEFAULT')],
+        ])
+
+        manager.to_sql.must_be_like %{
+          INSERT INTO \"users\" (\"id\", \"name\") VALUES ('1', 'david'), ('2', 'kirs'), ('3', DEFAULT)
```

Comment:
I don't think we should support that: Arel is a lower level library, so it tends to prefer to have one correct spelling, and less DWIM guessing. Let's just declare that this type expects an array of arrays.